src/overcooked_demo/README.md

```diff
@@ -58,12 +58,37 @@ If a more complex or custom loading routing is necessary, one can subclass the `
 
 ## Use the human vs. human game mode.
 
-With the Overcooked demo, you can test the interaction between two human players. To do this, you need to deploy this code on the server (https://docs.docker.com/language/python/deploy/). 
+### 1): Using Docker Compose CLI to deploy on EC2
+With the Overcooked demo, you can test the interaction between two human players. To do this, you need to deploy this code on the server (hsttps://docs.docker.com/language/python/deploy/). 
```

Comment:
```suggestion
With the Overcooked demo, you can test the interaction between two human players. To do this, you need to deploy this code on a server (hsttps://docs.docker.com/language/python/deploy/). 
```

"a" server? what do you mean by "the"?

Reply:
hmmm I didn't write this part. I think it was there on the old readme and I just copied and pasted it. I guess they are synonymous in this context but ..... I'll just change it.